From 7ed94b6860ad74291ae2dcb8f87495d941976ee9 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Fri, 19 Oct 2018 12:13:06 +0200
Subject: [PATCH] Merge android-9.0.0_r12

commit 732d274b836655263f86294024a9310e18137fd0
Merge: 6825a33 2efd888
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Sun Jun 24 03:06:03 2018 +0000

    Snap for 4859140 from 2efd888e7dc3b300ca98f7b2f6bb582fc96a44cd to pi-dr1-release

    Change-Id: I6dbe33b18c76c914ab3076061fb6e130fcc3ce10

commit 2efd888e7dc3b300ca98f7b2f6bb582fc96a44cd
Author: Bill Yi <byi@google.com>
Date:   Thu Jun 21 14:23:16 2018 -0700

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: Ice20c2f59609e7493b5dd4e7f16067bffdce0204

commit 6825a332e4170d4975d12ed1b5cdd2322fb3ff61
Merge: ef38496 f642544
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Thu Jun 7 03:00:37 2018 +0000

    Snap for 4826407 from f642544b499962ad93cad7b5f60c12244b0f6698 to pi-dr1-release

    Change-Id: Ia03e21f2a22ecf493112d0f63638f1c1f33df99c

Change-Id: Id52679cfa50f2c3161b26c51e446a3a161bcc29f
---
 res/values-ar/strings.xml | 2 +-
 res/values-as/strings.xml | 3 +--
 2 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/res/values-ar/strings.xml b/res/values-ar/strings.xml
index 4a8085c..1fc575f 100644
--- a/res/values-ar/strings.xml
+++ b/res/values-ar/strings.xml
@@ -59,7 +59,7 @@
     <string name="deletion_helper_no_threshold" msgid="6943179204098250444">"عرض كل العناصر"</string>
     <string name="deletion_helper_default_threshold" msgid="8410389370069021113">"إخفاء العناصر المستخدَمة حديثًا"</string>
     <string name="deletion_helper_clear_dialog_message_first_time" msgid="686530413183529901">"ستتم إزالة <xliff:g id="CLEARABLE_BYTES">%1$s</xliff:g> من المحتوى من جهازك."</string>
-    <string name="automatic_storage_manager_activation_warning" msgid="7657017408180001078">"تتم إدارة السعة التخزينية لجهازك الآن عن طريق مدير السعة التخزينية"</string>
+    <string name="automatic_storage_manager_activation_warning" msgid="7657017408180001078">"تتم إدارة سعة التخزين لجهازك الآن عن طريق مدير سعة التخزين"</string>
     <string name="empty_state_title" msgid="4033285438176545309">"ما من عناصر للإزالة"</string>
     <string name="empty_state_review_items_link" msgid="8411186441239304545">"مراجعة العناصر المستخدَمة حديثًا"</string>
     <string name="empty_state_summary" msgid="8439893007424243790">"ليست هناك أي ملفات قديمة لإزالتها. يمكنك إزالة الصور والفيديوهات والتطبيقات المستخدمة حديثًا من أجل إخلاء مساحة."</string>
diff --git a/res/values-as/strings.xml b/res/values-as/strings.xml
index d8da27d..742710d 100644
--- a/res/values-as/strings.xml
+++ b/res/values-as/strings.xml
@@ -19,8 +19,7 @@
     <string name="app_name" msgid="963247385700831716">"সঞ্চয়াগাৰ মেনেজাৰ"</string>
     <string name="cancel" msgid="7021218262867558825">"বাতিল কৰক"</string>
     <string name="storage_menu_free" msgid="1878247401436882778">"সঞ্চয়াগাৰত ঠাই খালী কৰক"</string>
-    <!-- no translation found for deletion_helper_title (3526170325226275927) -->
-    <skip />
+    <string name="deletion_helper_title" msgid="3526170325226275927">"বস্তুবোৰ আঁতৰাওক"</string>
     <string name="deletion_helper_app_summary" msgid="4904590040180275237">"<xliff:g id="DAYS">%1$d</xliff:g> দিনৰ পূর্বে"</string>
     <!-- no translation found for deletion_helper_app_summary_item_size (3770886184921427886) -->
     <skip />
-- 
2.17.1

